vuepress/docs/next/docs/releases/README.md

```diff
@@ -1,10 +1,197 @@
-# Entando 6.x Release Notes 
+
+# Entando 7.0.0 Release Notes
+
+## General
+
+Entando version 7.0 is a major release. It includes significant changes to the K8S infrastructure provisioning mechanism that provide better security, improved reliability and a simplified deployment process. Entando 7 introduces the [Entando Hub](../getting-started/landing-page.md#entando-hub), a rehauled compatibility matrix, development stack upgrades (e.g. JHipster 7.2) and several consolidation changes.
```

Comment:
Management is working on an updated summary of the value of 7.0 so I'm resisting rewriting this too much. One quick change, let's replace "rehauled compatibility matrix" with "updated compatibility matrix (e.g. Keycloak 15, PostgreSQL 14)".

Reply:
> It includes significant changes to the K8S infrastructure provisioning mechanism that provide better security, improved reliability and a simplified deployment process

and an extensibility mechanism based on the new capability provisioning workflow